src/lib.rs

```diff
@@ -1,5 +1,6 @@
 //! libm in pure Rust
 #![deny(warnings)]
+#![allow(unreachable_code)]
```

Comment:
This could be added on a file by file basis, but it shouldn't be at the crate root.